examples/24_gaussian_splats.py

```diff
@@ -84,7 +145,13 @@ def _(event: viser.GuiEvent) -> None:
         )
 
     for i, splat_path in enumerate(splat_paths):
-        splat_data = load_splat_file(splat_path, center=True)
+        if splat_path.suffix == ".splat":
+            splat_data = load_splat_file(splat_path, center=True)
+        elif splat_path.suffix == ".ply":
+            splat_data = load_ply_file(splat_path, center=True)
+        else:
+            sys.exit("Please provide a filepath to a .splat or .ply file.")
```

Comment:
nit can we replace this with a `raise SystemExit(...)` to avoid the extra import?